plugins/autosuspend.koplugin/main.lua

```diff
@@ -232,6 +232,8 @@ function AutoSuspend:onInputEvent()
     self.last_action_time = UIManager:getElapsedTimeSinceBoot()
 end
 
+AutoSuspend.onPageUpdate = AutoSuspend.onInputEvent
```

Comment:
I'd rather see an actual implementation, because right now the debug logging is wrong (and confusing).